ui/src/containers/BeamlineSetupContainer.jsx

```diff
@@ -296,13 +296,13 @@ class BeamlineSetupContainer extends React.Component {
           <Nav className="">
             <Nav.Item>
               <span className="blstatus-item">
-                {this.props.beamline.hardwareObjects.machine_info && (
+                {this.props.beamline.hardwareObjects.machine_info ? (
                   <MachInfo
                     info={
                       this.props.beamline.hardwareObjects.machine_info.value
                     }
                   />
-                )}
+                ) : null}
```

Comment:
Unrelated change? I prefer this change would not be introduced in this pull request.

Reply:
@marcus-oscarsson Was this a change required by the linter?

Reply:
No, not required. just nicer. Actually it should not even be there, ill remove it.

Reply:
Hm, actually its not nicer, just a mistake :)

Reply:
I see :D